bitnami/contour/values.yaml

```diff
@@ -667,6 +667,11 @@ envoy:
   ## Envoy Service properties
   ##
   service:
+    ## @param envoy.service.targetPorts [object] Map the controller service HTTP/HTTPS port
+    ##
+    targetPorts:
+      http: http
```

Comment:
Shouldn't these be numerical values?

Reply:
Nevermind! See: https://github.com/bitnami/charts/pull/9584#issuecomment-1087872902